TestRunner.psm1

```diff
@@ -376,41 +391,22 @@ function Invoke-TestRunner {
 
                 $TestType,
 
-                $TestRunnerModulePath,
-
                 [Parameter(Mandatory=$false)]
                 [PSCredential]
                 $credential = $null
             )
 
-            $commonFunctions = (Get-Command $TestRunnerModulePath).ScriptContents
-
             try {
                 $csvPath = "$runnerOutputPath/$TestType.$server.csv"
+                # Start with a fresh file
+                Remove-Item -Path $runnerOutputPath  -Include "$TestType.$server.csv" -Force  | Out-Null
 
-                # The Test-IsLocalhost function is not loading properly in the backgroun task
+                # The Test-IsLocalhost function is not loading properly in the background task
                 $isLocalhost = ($server.IndexOf("(local)") -gt -1) `
                     -or ($server.IndexOf("localhost") -gt -1) `
                     -or ($server.IndexOf(".") -eq 0) `
                     -or ($server.IndexOf($env:ComputerName) -gt -1)
 
-                if ($isLocalhost) {
-                    $actualPerformanceCounters = Compare-Object (typeperf -q) $expectedPerformanceCounters -PassThru -IncludeEqual -ExcludeDifferent
```

Comment:
The script never did anything with `actualPerformanceCounters`, so I removed this entire block of code.